suites/portal/discoveryPageTest.js

```diff
@@ -0,0 +1,98 @@
+const uuid = require('uuid');
+
+const I = actor();
+I.cache = {};
+
+Feature('Discovery page');
+
+Scenario('User is able to navigate to Discovery page @manual', ({ discovery }) => {
+  discovery.do.goToPage();
+  discovery.ask.isPageLoaded();
+});
+
+Scenario('Publish a study, search and export to workspace @manual', ({
```

Comment:
should we still leave the @manual tag there?
Is that still going to be a semi-automated test?

Reply:
Like I mentioned above, this is still WIP, so left it as manual for now to prevent it from running accidentally in CI. Once the feature is available in master, I will enable it in CI envs and remove the tag